dnainator-core/src/main/java/nl/tudelft/dnainator/parser/DRMutationParser.java

```diff
@@ -0,0 +1,32 @@
+package nl.tudelft.dnainator.parser;
+
+import nl.tudelft.dnainator.annotation.DRMutation;
+
+import java.io.IOException;
+import java.util.NoSuchElementException;
+
+/**
+ * An interface for parsing drug resistant mutations entries, using an Iterator pattern.
+ */
+public interface DRMutationParser {
```

Comment:
possibly, however, parsers like the `EdgeParser` follow a similar pattern. I did the same as we did there.